recipe/meta.yaml

```
@@ -6,16 +6,17 @@ package:
   version: {{ version }}
 
 source:
-  git_url: https://github.com/dgehringer/sqsgenerator.git
-  git_rev: v{{ version }}-rc.1
+  url: https://github.com/dgehringer/sqsgenerator/archive/refs/tags/v{{ version }}-rc.1.tar.gz
```

```suggestion
  url: https://github.com/dgehringer/sqsgenerator/archive/v{{ version }}-rc.1.tar.gz
```

Based on https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L21

---

Many thanks for pointing that out 👍